README.md

```diff
@@ -23,26 +24,26 @@ docker run -it releases-docker.jfrog.io/jfrog-ecosystem-integration-env
 
 The image is using `apt` and `sdkman` to download the build tools. Note: In the `:latest` tag, the tools versions may change.
 
-Operating system: Ubuntu 20.04.
+Operating system: ![GitHub](https://img.shields.io/static/v1?label=Ubuntu&message=+22.04+LTS&color=blue&style=for-the-badge&logo=ubuntu)
 
 |  Tool   | Current version |    Package name    |                      Source                      |
 | :-----: | :-------------: | :----------------: | :----------------------------------------------: |
-|  .NET   |  2.1.x, 3.1.x   | dotnet-sdk-2.1,3.1 | https://packages.microsoft.com/ubuntu/20.04/prod |
-|  cURL   |     7.68.x      |        curl        |                  Ubuntu archive                  |
-| Docker  |     20.10.x     |       docker       |     https://download.docker.com/linux/ubuntu     |
+|  .NET   |       6.0       |   dotnet-sdk-6.0   | https://packages.microsoft.com/ubuntu/22.04/prod |
+|  cURL   |     7.81.0      |        curl        |                  Ubuntu archive                  |
+| Docker  |     23.0.2      |       docker       |     https://download.docker.com/linux/ubuntu     |
 |   Go    |     1.20.x      |     golang-go      |              https://golang.org/dl               |
-| Gradle  |      7.2.x      |       gradle       |          https://sdkman.io/sdks#gradle           |
+| Gradle  |      8.0.2      |       gradle       |          https://sdkman.io/sdks#gradle           |
```

Comment:
I'm not sure this version is supported by our tools. Let's use Major 7 if 8 is not supported.